src/components/NavigationBar/index.js

```diff
@@ -10,10 +10,16 @@ const NavigationBar = ({back, translate, disabled, isFullScreen, className}) =>
   <div className={classNames(className, style.navigation, {
     [style.fullScreenNav]: isFullScreen
   })}>
-    <button href='#' className={classNames(style.back, {[style.disabled]: disabled, [style.backHoverDesktop]: isDesktop})}
-      onClick={preventDefaultOnClick(back)}>
+    <button
+      aria-label={translate('back')}
+      className={classNames(style.back, {
+        [style.disabled]: disabled,
+        [style.backHoverDesktop]: isDesktop
+      })}
+      onClick={preventDefaultOnClick(back)}
+    >
         <span className={style.iconBack} />
-        <span className={style.label}>
+        <span className={style.label} aria-hidden="true">
```

Comment:
Why do you need `aria-hidden` here? Is it because the text disappears on small screens?

Reply:
That's correct. So the decision here was to hide the actual copy within the span and add `aria-label` to button itself that is read by screen reader all the time.